ArduCopter/defines.h

```diff
@@ -94,7 +94,8 @@ enum tuning_func {
     TUNING_RATE_YAW_FF =                54, // body frame yaw rate controller FF term
     TUNING_RATE_MOT_YAW_HEADROOM =      55, // motors yaw headroom minimum
     TUNING_RATE_YAW_FILT =              56, // yaw rate input filter
-    TUNING_WINCH =                      57  // winch control (not actually a value to be tuned)
+    TUNING_WINCH =                      57, // winch control (not actually a value to be tuned)
+    SYSTEM_ID_MAG =                     58  // magnitude of the system ID signal
```

Comment:
if we could rename this "_MAG" to something else that might be nice. Like "MAGNITUDE" or something like that.

Reply:
Done.